.appveyor.yml

```diff
@@ -1,7 +1,9 @@
+image: Visual Studio 2022
```

Comment:
Defaults otherwise to 2015 if not set explicitly, which does not have a python 3.10...